<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#cccccc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>1001</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : white</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_paper">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Добавить лист бумаги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_plate">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>470</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Добавить тарелку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_del_body">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>470</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Удалть объект</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>470</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>180</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Сохранить файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_change">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>470</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stack">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Создать стопку</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lin_change">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>470</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_report">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black</string>
    </property>
    <property name="text">
     <string>Проверить на стабильность</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
